package-system/pyside2/package_windows.bat

```diff
@@ -0,0 +1,120 @@
+@echo off
+
+REM 
+REM Copyright (c) Contributors to the Open 3D Engine Project.
+REM For complete copyright and license terms please see the LICENSE at the root of this distribution.
+REM  
+REM SPDX-License-Identifier: Apache-2.0 OR MIT
+REM 
+REM 
+
+REM TEMP_FOLDER and TARGET_INSTALL_ROOT get set from the pull_and_build_from_git.py script
+
+echo TEMP_FOLDER=%TEMP_FOLDER%
+echo TARGET_INSTALL_ROOT=%TARGET_INSTALL_ROOT%
+
+SET PACKAGE_BASE=%TARGET_INSTALL_ROOT%
+
+SET INSTALL_SOURCE_RELEASE=%TEMP_FOLDER%\src\testenv3a_install\py3.10-qt5.15.1-64bit-release
+SET INSTALL_SOURCE_DEBUG=%TEMP_FOLDER%\src\testenv3dp_install\py3.10-qt5.15.1-64bit-debug
+SET BUILD_SOURCE_DEBUG=%TEMP_FOLDER%\\src\testenv3dp_build\py3.10-qt5.15.1-64bit-debug
+
+REM Copy the LICENSE and README files
+
+ECHO copy %TEMP_FOLDER%\src\LICENSE.FDL %PACKAGE_BASE%\
+copy %TEMP_FOLDER%\src\LICENSE.FDL %PACKAGE_BASE%\
+ECHO copy %TEMP_FOLDER%\src\LICENSE.GPLv3 %PACKAGE_BASE%\
+copy %TEMP_FOLDER%\src\LICENSE.GPLv3 %PACKAGE_BASE%\
+ECHO copy %TEMP_FOLDER%\src\LICENSE.GPLv3-EXCEPT %PACKAGE_BASE%\
+copy %TEMP_FOLDER%\src\LICENSE.GPLv3-EXCEPT %PACKAGE_BASE%\
+ECHO copy %TEMP_FOLDER%\src\LICENSE.LGPLv3 %PACKAGE_BASE%\
+copy %TEMP_FOLDER%\src\LICENSE.LGPLv3 %PACKAGE_BASE%\
+ECHO copy %TEMP_FOLDER%\..\LICENSES.TXT %PACKAGE_BASE%\
+copy %TEMP_FOLDER%\..\LICENSES.TXT %PACKAGE_BASE%\
+ECHO copy %TEMP_FOLDER%\src\README.* %PACKAGE_BASE%\
+copy %TEMP_FOLDER%\src\README.* %PACKAGE_BASE%\
+
+
+REM Copy the bin (release) folder 
+mkdir %PACKAGE_BASE%\bin
+robocopy %INSTALL_SOURCE_RELEASE%\bin %PACKAGE_BASE%\bin *.* /E
+REM Copy over libclang
+copy %TEMP_FOLDER%\libclang-release_130-based-windows-vs2019_64\libclang\bin\libclang.dll %PACKAGE_BASE%\bin\
```

Comment:
I see we copy it here, but... where do we copy or provide the license file?  Is it a permitted license?

Reply:
Good catch. 
```
LLVM System Interface Library
-------------------------------------------------------------------------------
The LLVM System Interface Library is licensed under the Illinois Open Source
License and has the following additional copyright:

Copyright (C) 2004 eXtensible Systems, Inc.
```
I believe this refers to the University of Illinois Open Source License which is on the pre-approved list ( https://opensource.org/licenses/UoI-NCSA.php )

Reply:
We should at least include that license, or put it in the same place as the files in the package, or etc. 
Basically we need to make sure everything is clear and disclosed.

Reply:
Will add that license file to the package

Reply:
The above license is now copied to LICENSE.LIBCLANG.TXT in the package